packages/react-data-grid/src/common/editors/EditorContainer.tsx

```diff
@@ -252,7 +252,7 @@ export default class EditorContainer extends React.Component<Props, State> {
       && inputNode.selectionStart === inputNode.value.length;
   }
 
-  handleRightClick = (e: MouseEvent<HTMLDivElement>) => {
+  handleRightClick = (e: React.MouseEvent<HTMLDivElement>) => {
```

Comment:
Had to fix this, as otherwise I get this error after tweaking the tsconfig:
```
> tsc

src/common/editors/EditorContainer.tsx:255:3 - error TS4031: Public property 'handleRightClick' of exported class has or is using private name 'MouseEvent'.

255   handleRightClick = (e: MouseEvent<HTMLDivElement>) => {
      ~~~~~~~~~~~~~~~~


Found 1 error.
```